src/components/StocksList.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { Sparklines, SparklinesLine } from 'react-sparklines';
+
+// StocksList component exported, used by App
+export const StocksList = (props) => {
+  console.log(`StocksList render: ${Date.now()}`);
+  return (
+    <div className="card column is-one-third" id="stocks_list">
+      <div className="card-header">
+        <div className="card-header-title">
+          <p className="stock-header-button">Stocks</p>
+          &nbsp;
+          <button className={(props.areUpdatesPaused() ? "tag is-danger" : "tag is-success")+" stock-header-button"}
```

Comment:
This button's styles look odd.
Remove classname 'tag', and use 'button is-small instead'